src/clientiface.h

```diff
@@ -402,7 +411,7 @@ class RemoteClient
 
 		List of block positions.
 	*/
-	std::set<v3s16> m_blocks_modified;
+	std::unordered_set<v3s16> m_blocks_modified;
```

Comment:
Offtopic: Someone should figure out some day whether keeping a modest number of position vectors is faster inside a `set` or a `vector`